suites/squid/rbd/tier-2_rbd_group.yaml

```diff
@@ -164,3 +164,24 @@ tests:
       module: test_rbd_clone_flatten_group_snapshot.py
       name: Test cloning and flattening group snapshot
       polarion-id: CEPH-83594299
+
+  - test:
```

Comment:
we need to keep scale in tier-3 suites can we add this here 
https://github.com/red-hat-storage/cephci/blob/master/suites/squid/rbd/tier-3_rbd_snap_operations.yaml

Reply:
Sure, Done